samigo/samigo-app/src/webapp/jsf/author/authorIndex_content.jsp

```diff
@@ -499,13 +498,17 @@
                         </h:panelGroup>
 
                         <h:panelGroup rendered="#{assessment.submittedCount>0 and (authorization.gradeAnyAssessment or authorization.gradeOwnAssessment)}">
-                            <h:commandLink title="#{authorFrontDoorMessages.t_score}" action="#{author.getOutcome}" immediate="true" id="authorIndexToScore1" >
-                                <h:outputText value="#{assessment.submittedCount}" />
-                                <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.evaluation.ResetTotalScoreListener" />
-                                <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.evaluation.TotalScoreListener" />
-                                <f:param name="publishedId" value="#{assessment.publishedAssessmentId}" />
-                                <f:param name="allSubmissionsT" value="3"/>
-                            </h:commandLink>
+                            <f:verbatim><span aria-label="</f:verbatim>
+                                <h:outputText value="#{assessment.submittedCount} #{authorFrontDoorMessages.assessment_submissions}" />
+                            <f:verbatim>" role="link" tabindex="0"></f:verbatim>
+                                <h:commandLink title="#{authorFrontDoorMessages.assessment_submissions}" action="#{author.getOutcome}" immediate="true" id="authorIndexToScore1" tabindex="-1">
+                                    <h:outputText value="#{assessment.submittedCount}" />
+                                    <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.evaluation.ResetTotalScoreListener" />
+                                    <f:actionListener type="org.sakaiproject.tool.assessment.ui.listener.evaluation.TotalScoreListener" />
+                                    <f:param name="publishedId" value="#{assessment.publishedAssessmentId}" />
+                                    <f:param name="allSubmissionsT" value="3"/>
+                                </h:commandLink>
+                            <f:verbatim></span></f:verbatim>
```

Comment:
I didn't found a better way to add an _aria-label_ to the _commandLink_ :/

Reply:
Well, I've seen similar approaches in Samigo, not ideal but it works fine. I usually do it via JS when the content loads but I don't know what's better..... thanks Vicent.